### PR TITLE
Audit logs

### PR DESCRIPTION
**Pull request by danielobyrne**
Adding documentation around audit logs to the cloud.

### CODE REVIEW

**Comment by danielobyrne** [comment]
Pushing to live now for sake of time. Then having Dale review when possible

File: source/audit-logs/activating-audit-logs.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+logging functionality for your projects.
+
+Prerequisites
+===============
+
+Before going further, you will need to ensure that you have :ref:`sourced an
+openRC file<source-rc-file>` for your command line so that you are able to
+perform the commands mentioned in the following example.
+
+Creating a container to house your logs
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:25]: Suggest changing to: "Create an object storage container for your logs."

File: source/audit-logs/activating-audit-logs.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+=======================================
+
+As the functionality of the audit log service allows you to aggregate logging
+from multiple projects into one place, you will need to choose a location that
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:29]: suggest changing "place" to "object storage container".

File: source/audit-logs/activating-audit-logs.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+a security hazard.
+
+The simplest way to create this account is to send an invitation to an existing
+user account on your project with `+object-storage` appended to their email.
+
+For example:
+
+.. image:: user-invite-object-store.png
+
+This will send an email to your existing account, but logically, on the cloud,
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:73]: This is confusing to me. It does not send an email to your existing Openstack account.

It sends the invitation to the same email inbox you might have already used (provided the receiver email server supports the `+` extension - most do).

File: source/audit-logs/activating-audit-logs.rst [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    | links      | {u'self': u'https://api.osppd.por.catalystcloud.nz:5000/v3/users/bf32a9a2c69e4d718022101e867cccec/credentials/OS-EC2/7d84281f4bc542b987ddbxxxxxxxxxxx'} |
+    | project_id | 033556c5979b4c1281xxxxxxxxxxxxxx                                                                                                                        |
+    | secret     | 100e767eeb7b48dcaf25xxxxxxxxxxxx                                                                                                                        |
+    | trust_id   | None                                                                                                                                                    |
+    | user_id    | bf32a9a2c69e4d71xxxxxxxxxxxxxxxx                                                                                                                        |
+    +------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Prepare a list of projects receive audit logs
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:102]: add: "to" in this title.

File: source/audit-logs/activating-audit-logs.rst [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+=============================================
+
+For this step, you will need to run the following command and save the UUID
+and name of the projects which you would like audit logs applied to. If you
+have multiple projects, you can still aggregate their logs into the single
+container you created a the start of this tutorial.
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:108]: typo: "at"

File: source/audit-logs/activating-audit-logs.rst [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+    | trust_id   | None                                                                                                                                                    |
+    | user_id    | bf32a9a2c69e4d71xxxxxxxxxxxxxxxx                                                                                                                        |
+    +------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Prepare a list of projects receive audit logs
+=============================================
+
+For this step, you will need to run the following command and save the UUID
+and name of the projects which you would like audit logs applied to. If you
+have multiple projects, you can still aggregate their logs into the single
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:107]: Remove "still", redundant.

File: source/audit-logs/activating-audit-logs.rst [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+
+.. code-block:: bash
+
+    $ openstack project list
+
+Send this information through to the Catalyst Cloud Team
+==========================================================
+
+At this stage since this service is still in a technical preview, the final
+step needs to be preformed from the operations team. This means that you will
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:118]: replace "from" with "by"

File: source/audit-logs/activating-audit-logs.rst [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+   needed.
+#. The list of projects that you want to start receiving log data for.
+
+.. warning::
+
+    Do not include any information that would be sensitive or compromising,
+    such as the ``secret`` in the output of the ec2 credentials command.
+
+Once you have sent off your information to the Catalyst Cloud team and received
+confirmation, you should start to see .json log files appear in your
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:133]: Remove ".json" - the uploaded objects are not named ".json"

File: source/audit-logs/activating-audit-logs.rst [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+
+For this step, you will need to create a new user account for your project
+that only has the `Object Storage` role assigned. This is to restrict the
+permissions of said user to only the object storage service as the EC2
+credentials which are required for this step otherwise have a much larger reach
+into the project than is required to capture the logs; and would therefore be
+a security hazard.
+
+The simplest way to create this account is to send an invitation to an existing
+user account on your project with `+object-storage` appended to their email.
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:67]: your example screenshot differs from this text example. Remove '-'?

File: source/audit-logs/activating-audit-logs.rst [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+credentials for your account in order to access the necessary log data.
+
+For this step, you will need to create a new user account for your project
+that only has the `Object Storage` role assigned. This is to restrict the
+permissions of said user to only the object storage service as the EC2
+credentials which are required for this step otherwise have a much larger reach
+into the project than is required to capture the logs; and would therefore be
+a security hazard.
+
+The simplest way to create this account is to send an invitation to an existing
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:66]: Perhaps change "The simplest way" to "A convenient way".

File: source/audit-logs/activating-audit-logs.rst [hunk 11]
```diff
@@ -0,0 +1,134 @@
+###############################
+Log creation and access
+###############################
+
+.. warning::
+
+    The following service is still in a technical preview and as such, will have
+    limitations on features and scope. Any feedback is appreciated and can be
```
Comment (dalees) [inline source/audit-logs/activating-audit-logs.rst:8]: Q: Do we want to list the services and/or API actions that are supported in Audit Log, and add it to over time?

Comment (danielobyrne) [reply source/audit-logs/activating-audit-logs.rst:8]: I think that would be a good idea yeah. List the limitations so customers know what to expect.

**Pull request merged**